### PR TITLE
Remove try catch in test_torchinductor

### PR DESCRIPTION
**Pull request by eellison**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #93005
* __->__ #93004
* #93003

I think this was holdover compat code from multiple repros. we should error on failure. 

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: test/inductor/test_torchinductor.py [hunk 1]
```diff
@@ -35,42 +37,34 @@
...
+    permute_matmul_fusion,
+    sink_cat_after_pointwise,
+    transpose_linear,
+    transpose_matmul,
+)
+from torch._inductor.sizevars import SizeVarAllocator
+from torch._inductor.utils import has_torchvision_roi_align, timed
+from torch.fx.experimental.symbolic_shapes import FloorDiv
+
+# This will only pass on pytorch builds newer than roughly 5/15/2022
```
Comment (ngimel) [inline test/inductor/test_torchinductor.py:64]: nit: I think we can remove this

**Review by ngimel — APPROVED** [review]

**Comment by eellison** [comment]
Looks like windows fails when we remove the try catch...

**Comment by ngimel** [comment]
We can skip tests on windows for now, it's a pre-existing condition?

**Pull request closed without merging**